packages/eslint-plugin/rules/no-direct-date.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
 			NewExpression( node ) {
 				if (
 					node.callee.name === 'Date' &&
-					node.arguments.length === 0
+					node.arguments.length <= 2
```

Comment:
I usually find the "less than or equal to" easier to scan, because I don't have to do _complex arithmetic_ in my head. With `< 3` I need to calculate something, but for `<= 2` I can just go "oh, it's two or less". 😅

Reply:
I usually find the "less than or equal to" easier to scan, because I don't have to do _complex arithmetic_ in my head.

With < 3 I need to calculate something, but for <= 2 I can just go "oh, it's two or less". 😅

Reply:
> but for `<= 2` I can just go "oh, it's two or less".

I don't feel super strongly about it, so happy to keep what you have. Removing the "or" from that thought is what I mean by logically simpler.